### PR TITLE
Fix Issues 38 and 41 

### LINKED ISSUE #38: Make header text Bold in Build Summary table
Title

### LINKED ISSUE #41: Consider center aligning the summary header names
The summary header names seem to be left aligned. Is that intentional? If not, does it make sense to center-align them. And is it possible to bold that header row or make it standout as a header row.

### PR DESCRIPTION
**Pull request by nbhoski**
This PR closes #38 and closes #41 

This PR consist following changes 
* Table headers are now bold and center-aligned 

**Old table**
image

**New Table**
image

### CODE REVIEW

File: src/buildSummary.ts [hunk 1]
```diff
@@ -37,16 +21,31 @@ export function writeSummary(taskSummaryTableRows: string[][]) {
     }
 }
 
+export function getSummaryRows(buildSummary: string): any[] {
+    const rows = JSON.parse(buildSummary).map((t: { name: any; failed: { toString: () => any; }; skipped: { toString: () => any; }; description: any; duration: { toString: () => any; }; }) => {
```
Comment (sameagen-MW) [inline src/buildSummary.ts:25]: Is the type annotation needed here? If we can get away with omitting it I think that might be more readable in this case.

File: src/buildSummary.ts [hunk 2]
```diff
@@ -37,16 +21,31 @@ export function writeSummary(taskSummaryTableRows: string[][]) {
...
+        if (t.failed.toString() === 'true') {
```
Comment (sameagen-MW) [inline src/buildSummary.ts:26]: Do we need to convert to a string here?

**Review by sameagen-MW — APPROVED** [review]

File: src/buildSummary.ts (hunk 1 above)
Comment (nbhoski) [reply src/buildSummary.ts]: it seems atleast one annotation should be there. else need to create a interface. i have changed the entire line to just `(t: any)`

File: src/buildSummary.ts (hunk 2 above)
Comment (nbhoski) [reply src/buildSummary.ts]: :) No changed it

**Pull request merged**